src/transformers/models/audio_spectrogram_transformer/configuration_audio_spectrogram_transformer.py

```diff
@@ -0,0 +1,126 @@
+    architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+    Args:
+        hidden_size (`int`, *optional*, defaults to 768):
+            Dimensionality of the encoder layers and the pooler layer.
+        num_hidden_layers (`int`, *optional*, defaults to 12):
+            Number of hidden layers in the Transformer encoder.
+        num_attention_heads (`int`, *optional*, defaults to 12):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        intermediate_size (`int`, *optional*, defaults to 3072):
+            Dimensionality of the "intermediate" (i.e., feed-forward) layer in the Transformer encoder.
+        hidden_act (`str` or `function`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
+            `"relu"`, `"selu"` and `"gelu_new"` are supported.
+        hidden_dropout_prob (`float`, *optional*, defaults to 0.1):
+            The dropout probability for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (`float`, *optional*, defaults to 0.1):
+            The dropout ratio for the attention probabilities.
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        layer_norm_eps (`float`, *optional*, defaults to 1e-12):
+            The epsilon used by the layer normalization layers.
+        patch_size (`int`, *optional*, defaults to `16`):
+            The size (resolution) of each patch.
+        qkv_bias (`bool`, *optional*, defaults to `True`):
+            Whether to add a bias to the queries, keys and values.
+        frequency_stride (`int`, *optional*, defaults to 10):
+            Frequency stride to use when patchifying the spectrograms.
+        time_stride (`int`, *optional*, defaults to 10):
+            Temporal stride to use when patchifying the spectrograms.
+        time_dimension (`int`, *optional*, defaults to 1024):
+            Temporal dimension of the spectrograms.
+        frequency_dimension (`int`, *optional*, defaults to 128):
+            Frequency dimension of the spectrograms.
+
+    Example:
+
+    ```python
+    >>> from transformers import AudioSpectrogramTransformerModel, AudioSpectrogramTransformerConfig
+
+    >>> # Initializing a AudioSpectrogramTransformer audio_spectrogram_transformer-base-patch16-224 style configuration
+    >>> configuration = AudioSpectrogramTransformerConfig()
+
+    >>> # Initializing a model from the audio_spectrogram_transformer-base-patch16-224 style configuration
```

Comment:
You should use the new version for the config docstring here (look at BERT for an example).